### DISCUSSION TITLE
Calculate "etxra charge" of custom field only once in multibooking forms

### DISCUSSION THREAD

**Discussion by Schneleed** [liedekef/events-made-easy#471] [Q&A]
Dear franky,

I am currently setting up a multi booking scenario where I want to solve the following issue:

There is the need of booking a special insurance but the surcharge has only to be paid once per multibooking. When I create the custom field using the option "extra charge" the value is added to every booking in the multi booking form. Is there a simnple way to add a specific amount of money visible to the customer within the multibooking form as surcharge on the total price?

Thanks in advance for your help.

**Comment by liedekef** [comment]
There's none I can think about currently. Multibooking is in fact creating different bookings (and the total price is the sum of each individual booking), so the current behaviour is consistent (I don't know for which event the extra charge should be done).
I'm thinking about adding the extra_charge as an option for custom fields of type people too, but the problem is: what if it is already paid for then (and used in another form). Currently the best option is to look into the available hooks/filters for multibookings and adjust the price accordingly.

**Reply by liedekef** [reply]
(btw: moved to discussion since this is not an issue)

**Reply by Schneleed** [reply]
Short Update:
I redesigned the whole thing back to a multiseat event so I am able to use the "extra charge" option for my custom field as designed.

Another topic came up by redesigning it:
Is there already an idea to make a discount selectable by using radio buttons or a dropdown field? This would help tremendously to build multiseat events where I could give a certain differentiated pricing per seat by using a discount being applied e.g. a discount called children or adult to diversify based on age.

Thx for your thoughts on this!

**Reply by liedekef** [reply]
Multiprice is meant to create different price categories for different seat categories ...
If that's not enough, create a hidden field with extra negative charge, add it to a template and use dynamic data to show that template based on your conditions (it won't show the field since it is hidden but it will be taken into account for price calc then).
